lib/plausible_web/live/goal_settings/list.ex

```diff
@@ -26,6 +27,9 @@ defmodule PlausibleWeb.Live.GoalSettings.List do
         >
           Add Goal
         </.button>
+        <.button :if={not connected?(@socket)} id="add-goal-button" mt?={false}>
```

Comment:
Shouldn't that be

```suggestion
        <.button :if={connected?(@socket)} id="add-goal-button" mt?={false}>
```

instead? 🤔

Reply:
:man_facepalming:

Reply:
lmao github broke - original code:

![image](https://github.com/user-attachments/assets/7fc8785c-c42e-4507-a3e8-ca88edca294b)

Reply:
I take that back, the condition was valid, I have not connected the dots that it's a button in _inactive_ state 🤡